packages/js/package.json

```diff
@@ -54,8 +54,6 @@
     "bignumber.js": "^9.0.2",
     "bn.js": "^5.2.0",
     "bs58": "^5.0.0",
-    "buffer": "^6.0.3",
-    "cross-fetch": "^3.1.5",
```

Comment:
Same here

Reply:
not used/imported/etc

Reply:
Hmm, these are definitely used.

E.g. https://github.com/metaplex-foundation/js/blob/63c368aa9b1bbe1e86341f603e5de1bcc56eaaa9/packages/js/src/plugins/storageModule/MetaplexFile.ts#L1

`cross-fetch` should be used but it looks like it's usage was removed after a refactoring I made so I'll fix that when I have a sec.

Reply:
I think dephceck decided in favor of native node buffer. Fixed.